### PR TITLE
Update: Sigil-Ebook.Sigil version 1.7.0

### PR DESCRIPTION
**Pull request by OfficialEsco**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you validated your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23907)

### CODE REVIEW

File: manifests/s/Sigil-Ebook/Sigil/1.7.0/Sigil-Ebook.Sigil.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Sigil-Ebook.Sigil
+PackageVersion: 1.7.0
+PackageLocale: en-US
+Publisher: Sigil-Ebook
+PublisherUrl: https://sigil-ebook.com/
+PublisherSupportUrl: https://sigil-ebook.com/support
```
Comment (ImJoakim) [inline manifests/s/Sigil-Ebook/Sigil/1.7.0/Sigil-Ebook.Sigil.locale.en-US.yaml]:
```suggestion
PublisherSupportUrl: https://sigil-ebook.com/sigil/support/
```

Comment (ImJoakim) [reply manifests/s/Sigil-Ebook/Sigil/1.7.0/Sigil-Ebook.Sigil.locale.en-US.yaml]: Whoops, nevermind!

Comment (OfficialEsco) [reply manifests/s/Sigil-Ebook/Sigil/1.7.0/Sigil-Ebook.Sigil.locale.en-US.yaml]: Uuf

**Comment by OfficialEsco** [comment]
Hmm
image

**Comment by OfficialEsco** [comment]
@KaranKad any idea on why this suddenly is a issue? Its also appears on version 1.6.0 now which passed both manual and automatic validation 2021-06-08

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**